packages/shopify-cli-extensions/src/configs.ts

```diff
@@ -87,7 +87,7 @@ function jsonConfigs<T extends Indexable>(
   paths: string[] = [],
   formatter = toCamelCase,
 ): T {
-  return Object.keys(configs).reduce((acc, key) => {
+  return Object.keys(configs || {}).reduce((acc, key) => {
```

Comment:
Same question here - why do we need a fallback?

Reply:
I believe all 3 of these fallbacks were needed when we were testing it.